index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Alice Cieslik . Persönliche Website</title>
+  </head>
+  <body>
+    <header>
+      <p><strong>Alice Cieslik . Persönliche Website</strong> </p>
```

Comment:
```suggestion
      <h1>Alice Cieslik . Persönliche Website</h1>
```
I think this is the main heading for your website. Therefore, I would recommend you to use the `h1` element.